README.md

```diff
@@ -1,2 +1,2 @@
 # pr-test
-Test the PR test flow
+Test the pull request test flow
```

Comment:
please remove the word 'test' at the end. The sentence should be 'Test the pull request flow'

Reply:
updated the sentence